utils/generate_types/test/tsconfig.json

```diff
@@ -3,7 +3,14 @@
     "strict": true,
     "target": "ES2019",
     "noEmit": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "baseUrl": "../../..",
+    "paths": {
```

Comment:
Hmm... It would be better for this test to have a setup like the end user would.